### PR TITLE
Added Pages and basic UI

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Pages and basic UI

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,14 +2,22 @@ import React from "react";
...
+import { CartContext } from "./context/cart-context.js";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <CartContext.Provider value={{ items: 5 }}>
```
Comment (lannisterrr) [inline src/index.js:17]: You can also create a component that returns this context. Something like `<CartProvider></CartProvider>`

File: src/pages/cart.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                </button>
+                <button class="btn-icon btn-link">
+                  <span> Move to Wishlist </span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </figure>
+      </aside>
+      <main class="products-display grow">
```
Comment (lannisterrr) [inline src/pages/cart.js:43]: this can be a seperate component.

**Review by lannisterrr — COMMENTED** [review]
clean code , some suggesstions made.

**Review by abhij1607 — COMMENTED** [review]
Clean Code | | Good practices followed | |  try to break pages into smaller components in future functionality implementation

File: src/index.js (hunk 1 above)
Comment (nareshpoonia) [reply src/index.js:17]: Thanks for the comment siddhant, I'll update the component, when I'll use this context for real functionality.
It was for simply testing it

File: src/pages/cart.js (hunk 2 above)
Comment (nareshpoonia) [reply src/pages/cart.js:43]: Thanks for the suggestion, I'll be breaking down my code into components as I moved ahead with the full project

**Pull request merged**